recipes/libxml2/all/conanfile.py

```diff
@@ -206,35 +215,65 @@ def package(self):
         self.copy("COPYING", src=self._source_subfolder, dst="licenses", ignore_case=True, keep_path=False)
         if self._is_msvc:
             self._package_msvc()
+            # remove redundant libraries to avoid confusion
+            if not self.options.shared:
+                os.remove(os.path.join(self.package_folder, "bin", "libxml2.dll"))
+            os.remove(os.path.join(self.package_folder, "lib", "libxml2_a_dll.lib"))
+            os.remove(os.path.join(self.package_folder, "lib", "libxml2_a.lib" if self.options.shared else "libxml2.lib"))
```

Comment:
Could this be simplified?

```suggestion
            tools.remove_files_by_mask(os.path.join(self.package_folder, "lib"), "libxml2*.lib")
```

Reply:
No. You remove both `libxml2_a.lib` and `libxml2.lib`. One must remain.